examples/step-33/step-33.cc

```diff
@@ -201,14 +201,14 @@ namespace Step33
     }
 
```

Comment:
> 在 2015年4月20日，下午1:46，Martin Kronbichler <notifications@github.com> 写道：
> 
> In examples/step-33/step-33.cc <https://github.com/dealii/dealii/pull/851#discussion_r28716939>:
> 
> > @@ -201,14 +201,14 @@ namespace Step33
> >      }
> >  
> There is one thing one could address with this code: deal.II vector now also provide an operator[], so that code could be simplified. Do you want to try? (I will merge this PR anyway because it is unrelated to the proposed patch.)
> 
> 

No problem. That is not hard but can make the code much more pretty.